packages/docgen/test/get-type-annotation.js

```diff
@@ -263,4 +264,36 @@ describe( 'Type annotations', () => {
 			).toBe( '( foo: string, ...rest: any[] ) => GenericType< T >' );
 		} );
 	} );
+
+	describe( 'function argument object destructuring', () => {
+		describe( 'type literal', () => {
+			const node = getObjectDestructuringTypeLiteralNode();
+
+			it( 'should get the full type for the param', () => {
+				expect( getTypeAnnotation( paramTag, node, 0 ) ).toBe(
```

Comment:
What is `paramTag` in the context of these tests?

Reply:
`paramTag` is defined once at the top of the test file, it represents the `comment-parser` AST for the `@param` tag in the JSDoc/TSDoc.